qa_forum/static/js/main.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+    $('.comment_submit').on('click', function() {
+        var raiser = $(this).prev();
+        var comment = $(this).parent().prev().children();
+        var pointTo = $(this).val();
+        if (raiser.val() && comment.val()) {
+            send_comment(raiser, comment, pointTo);
+        } else {
+            alert("Please fill in both your name and the comment")
+        }
+        clear(raiser, comment);
+    });
+
+    function clear(raiser, comment){
+        raiser.val("");
+        comment.val("");
+    }
+
+    function send_comment(raiser, comment, pointTo) {
+        var data = {
+            "raiser": raiser.val(),
+            "comment": comment.val(),
+            "pointQ": pointTo
+        };
+        $.ajax({
+            url: '/answer_form/',
+            type: 'POST',
+            dataType: 'json',
+            data: JSON.stringify(data),
+            success: function() {
+                window.location.href = "/";
```

Comment:
create a new DOM node  --> That's a good point and it's what I intended to do, but I added a notification if there is no comment right now. So there should be an additional if statement to check if there is the notification and remove it. [window.location.href = "/";] just want to make it simple, but surely your point is right.

if the POST request fails  --> Yes, I should add a statement
failure: function() {
     alart(....)
}